CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Networking/DocumentService.swift

```diff
@@ -14,11 +14,36 @@ public final class DocumentService: DocumentServiceProtocol {
     public var document: Document?
     public var analysisCancellationToken: CancellationToken?
     public var metadata: Document.Metadata?
-    var documentService: DefaultDocumentService
+    
+    var captureNetworkService: GiniCaptureNetworkService
     
     public init(lib: GiniBankAPI, metadata: Document.Metadata?) {
         self.metadata = metadata
-        self.documentService = lib.documentService()
+        self.captureNetworkService = DefaultCaptureNetworkService(lib: lib)
```

Comment:
Nice! Very elegantly solved to keep the internal changes hidden from the outside and avoid changing existing code. 👏